File: pattern2.c
#include<stdio.h>
int main()
{
int n,i,j,k;
printf("Enter no.of rows\n");
scanf("%d",&n);
for(i=n;i>1;i--)
{
    for(j=i;j<=n;j++)
    {
        printf(" ");
    }
    for(k=0;k<(2*i)-1;k++)
    {
        printf("*");
    }
    printf("\n");
}
    for(i=1;i<=n;i++)
{
    for(j=0;j<=n-i;j++)
    {
        printf(" ");
    }
    for(k=0;k<(2*i)-1;k++)
    {
        printf("*");
    }
    printf("\n");
}
    printf("\n");
    return 0;

}

File: pattern2.py
n=int(input("Enter the no.of rows"))
for i in range (1,n+1):
    for j in range(1,n-i):
        print('',end='')
    for j in range(1,i+1):
            print("*",end='')
    print()


rows = int(input("Please Enter the Total Number of Rows  : "))

print("Mirrored Right Triangle Star Pattern")
for i in range(1, rows + 1):
    for j in range(1, rows + 1):
        if(j <= rows - i):
            print(' ', end = '  ')
        else:
            print('*', end = '  ')
    print()


n=int(input("Enter the no.of rows"))
for i in range (n):
    for j in range(i,n):
        print('*',end=' ')
    print()


n=int(input("Enter the no.of rows"))
for i in range (n):
    for j in range(i,n):
        print(' ',end=' ')
    for j in range(i+1):
        print('*',end=' ')
    print()

